doc/other_modules/tiled.md

```diff
@@ -1,56 +1,10 @@
 # Tiled
 
-[Tiled](https://www.mapeditor.org/) is an great tool to design levels and maps.
+[Tiled](https://www.mapeditor.org/) is a great tool to design levels and maps.
 
-Flame provides a package ([flame_tiled](https://github.com/flame-engine/flame_tiled)) which bundles
-a [dart package](https://pub.dev/packages/tiled) that allows you to parse tmx (xml) files and access
-the tiles, objects and everything in there.
+Flame provides a package ([flame_tiled](https://github.com/flame-engine/flame_tiled)) that
+bundles a [dart](https://pub.dev/packages/tiled) package which
+allows you to parse TMX (XML) files and access the tiles, objects, and everything in there.
 
 Flame also provides a simple `Tiled` class and its component wrapper `TiledComponent`, for the map
 rendering, which renders the tiles on the screen and supports rotations and flips.
-
-
-## Layers
```

Comment:
Where has this moved?

Reply:
Ok, this whole file was very confusing because it references sections from `flame_tiled` package.  The example is even called `flame_tiled` yet has nothing to do with that Bridge Library.  I am pulling some sections back, but the example should be renamed as it is confusing.

Reply:
The `TiledComponent` exist in `flame_tiled` and the `Tiled` class exist in the `tiled` library.
Other modules that we maintain could be for example AudioPlayers, Forge2D, Tiled and Oxygen, that's where the most of the functionality lies and they can be used without Flame, and then we have the bridge packages that creates components for using that functionality. We probably should organize this better though since even you get confused about it.

Reply:
Ok, then this section should really be about the Tiled library.  All the information about `TileComponent` etc should be in the bridge package section (which already exists).  Does AudioPlayers, Tiled, Oxygen, etc have their own documentation page outside of Flame?

Reply:
Maybe we should just have the bridge libraries section and then we can add subsections there that links and/or explains things for the library that it bridges to. 

Tiled should also link the the tiled editor and their docs for example and not only to the dart port.

Reply:
Can do.

Reply:
Ok, I have adjusted Tiled and went ahead and grab some content from the Tiled docs to explain what Tiled does better. Moved it all to the Flame tiled bridge package, but game a specific callout to Tiled.  There are some prior merge issues that were resolved as well.

Reply:
I think that it sounds good to include an image of the Tiled editor!

Reply:
Will do.

Reply:
Done.